clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -25,14 +20,14 @@ import shapeless.CNil
   */
 sealed trait ClioCommand
 
-case object GetWgsUbamSchema extends ClioCommand
+case object GetSchemaWgsUbam extends ClioCommand
```

Comment:
This will change the command name to `'get-schema-wgs-ubam'`. It's hard-coded in the add executor now, so it should be updated.

Reply:
Done (in executor)